src/univariates.jl

```diff
@@ -369,6 +369,24 @@ logccdf(d::UnivariateDistribution, x::Real) = log(ccdf(d, x))
 Evaluate the inverse cumulative distribution function at `q`.
 
 See also: [`cquantile`](@ref), [`invlogcdf`](@ref), and [`invlogccdf`](@ref).
+
+# Examples
+```jldoctest
+julia> d = Normal()
+Normal{Float64}(μ=0.0, σ=1.0)
+
+julia> first_quartile = quantile(d, 0.25)
+-0.6744897501960818
+
+julia> second_quartile = quantile(d, 0.50)
+0.0
+
+julia> third_quartile = quantile(d, 0.75)
+0.6744897501960818
+
+julia> quantile(d, 0.025), quantile(d, 0.975) # 95%-probability interval 
+(-1.9599639845400592, 1.9599639845400576)
+```
```

Comment:
Maybe add:

````suggestion

See also [`median`](@ref)
```
````
And possibly also use it in the doctests, something like `quantile(d, 0.5) ≈ median(d)`.

Reply:
Sounds like a good idea. I am not sure how doctests work - can I include `julia>` and the output without problem, just like I have?